src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.*;
```

Comment:
Should only import classes that are necessary for your code to avoid creating conflicting methods

Reply:
Thanks for that!